actioncable/jstest/unit/action_cable.js

```diff
@@ -0,0 +1,49 @@
+require('../test_helper');
+
+describe('ActionCable', function() {
```

Comment:
Agree, to reflect CS counterparts. https://github.com/basecamp/trix/tree/master/test/src are good examples for these.